src/ParticipationStorage/ParticipationStorageInterface.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Phpab\Phpab\Storage;
+
+/**
+ * Stores the participation state of the user
+ */
+interface ParticipationStorageInterface
+{
+    /**
+     * The identifier the session/cookie should be stored in.
+     *
+     * @return string
+     */
+    public function getIdentifier();
```

Comment:
I don't think this should be part of the interface. For storage-X this might not be relevant at all. Also, in case of a CookieStorage for example, this will be encapsulated in that class.